layers/headers.go

```diff
@@ -19,7 +19,7 @@ func (h *XHeadersLayer) OnRequest(state *httransform.LayerState) error {
 		state.RequestHeaders.SetString(k, v)
 	}
 
-	profile, ok := state.RequestHeaders.GetString("x-crawlera-profile")
+	profile, ok := state.RequestHeaders.GetString("zyte-proxy-profile")
```

Comment:
Is **zyte-proxy-profile** implemented? I seem we have to keep **x-crawlera-profile**.